### PR TITLE
Changes for 2018

### PR DESCRIPTION
**Pull request by rkost**
**Before merge:**

- [x] 2018 needs to be set to default page
- [x] Talks need to be scheduled / listed (done with placeholders)
- [x] Add ideas for workshops
- [ ] Rework press section
  - [ ] rework text as there is no public day
  - [ ] invite the press via the website¿
- [x] Sponsors need to be updated
  - [x] Add new sponsors
  - [x] Remove old sponsors
  - [x] Put in correct order¿¡
- [x] European Robotics week needs to be removed or updated
- [x] Remove teams from the registration page
- [x] Search for any wrong dates
  - [x] Insert correct deadlines in `/en/teams.md` - needs to be consistent with the mail.
- [x] Better links to the recommended hotels. Some links only point to the main page of the brand.
- [x] Search for any wrong announcements (soccer games will not be there this year)
- [x] Review `Rechschreibung` and `Grammatik` again

Feel free to add todos...

### CODE REVIEW

**Comment by ploth** [comment]
Would merge and move task list to extra issue.

**Comment by ploth** [comment]
When the website is somehow ready we can change the default page.

**Comment by ploth** [comment]
Hmm, how about a english only website. Way easier to maintain

**Comment by ploth** [comment]
haha

File: 2018/de/visitors.md [hunk 1]
```diff
@@ -3,25 +3,7 @@ title: Informationen für Besucher
 layout: default-de-2018
 ---
 
-## Austragungsort
```
Comment (ploth) [inline 2018/de/visitors.md]: Would not remove the address.

File: 2018/en/schedule.md [hunk 2]
```diff
@@ -3,62 +3,48 @@ title: Schedule
 layout: default-en-2018
 ---
 
-## Friday (1<sup>st</sup> December 2017, internal)
+## Friday (30<sup>th</sup> November 2018, internal)
```
Comment (ploth) [inline 2018/en/schedule.md]: remove internal. its internal in any case and don't raises false hopes.

Comment (rkost) [reply 2018/en/schedule.md]: Will do, thanks

**Comment by rkost** [comment]
> Would merge and move task list to extra issue.

Haha, did not see your reply. Will finish the website until tomorrow evening anyway, so I will keep the PR for now

**Comment by rkost** [comment]
Current state of the sponsor banner: 

image

**Comment by ploth** [comment]
* IAV is not part of the sponsoring yet.
* order: eppendorf, Ibeo, freiheit.com, Sick AG, Alfons Haar, Basler, TUHH

**Comment by rkost** [comment]
image

**Pull request merged**